src/test/java/hudson/tasks/AntTest.java

```diff
@@ -123,6 +124,12 @@ private void verify() throws Exception {
         assertEquals(1,l.length);
         r.assertEqualBeans(l[0],new AntInstallation("myAnt","/tmp/foo", JenkinsRule.NO_PROPERTIES),"name,home");
 
+        // Verify that PATH+ANT is set.
```

Comment:
That is not what we actually care about. We want to know if a full environment, once `expand`ed, contains a `PATH` which starts with the new dir.